src/CMakeLists.txt

```diff
@@ -120,6 +120,8 @@ set( MIOpen_Source
     solver/activ/fwd_1.cpp
     solver/activ/bwd_0.cpp
     solver/activ/bwd_1.cpp
+    batchnorm/problem_description.cpp
+    solver/batchnorm/forward_per_activation_single.cpp
```

Comment:
Only forward batchnorm code reorg and other changes are done.  How about backward batchnorm?

Reply:
I make one solver per PR. Otherwise, this would be even more immense. This one is focused on the forward single kernel batchnorm.